### DISCUSSION TITLE
QuteBrowser not launching

### DISCUSSION THREAD

**Discussion by raghavken** [qutebrowser/qutebrowser#5911] [Q&A] [answered]
I just recently installed qutebrowser and when i launch it, it closes, after attempting to launch it from the terminal the following logs were seen.

11:37:07 CRITICAL: Cannot mix incompatible Qt library (5.15.2) with this library (5.15.1)
[1]    5670 abort (core dumped)  qutebrowser

**Comment by rien333** [comment]
How did you install qutebrowser? Seems like the qt library it was build against is older than the one on your system.

**Reply by raghavken** [reply]
I installed qutebrowser via pacman

**Reply by rien333** [reply]
Okay, odd. But as the The-Compiler wrote:

> Like the error message says, you have a mixture between Qt 5.15.2 and 5.15.1 on your system. 

How did you install Qt? Do you have any other Qt installations on your system? Are you sure your system is fully updated?

**Comment by The-Compiler** [comment, marked as answer]
Like the error message says, you have a mixture between Qt 5.15.2 and 5.15.1 on your system. It's likely that any other Qt application would fail in the same way.

You might be able to run with `QT_DEBUG_PLUGINS=1` to see what plugin Qt loads, and then somehow (e.g. via your package manager) find out what of it is outdated. If you're on Archlinux, Qt (style) plugins you installed from the AUR but didn't recompile are likely candidates.
